tests/dummy/app/styles/_variables.css

```diff
@@ -0,0 +1,34 @@
+/* Colors */
+
+:root {
```

Comment:
I believe CSS variables here without PostCSS might kinda break the demo site on IE11, but I think that's probably fine.

Reply:
I hope that the second commit should help with this — the variables now only override the default colour scheme so on IE it should display OK but without code highlighting.

Reply:
Am fine with using css variables. I'd be surprised if any devs are using IE11 😛